test/basic.html

```diff
@@ -72,88 +63,59 @@
       });
 
       test('can render native elements', function() {
-        var element = document.getElementById('nativeDemo');
+        Polymer.Base.async(function() {
```

Comment:
why this async? can you put a comment in here?
Also, even if this needs to be async, it's better to have a single `suiteSetup` rather than having each test wait for 1ms.
PS: you made these tests async but forgot to call `done()` at the end :)